applications/RomApplication/custom_utilities/rom_auxiliary_utilities.h

```diff
@@ -107,6 +107,34 @@ class KRATOS_API(ROM_APPLICATION) RomAuxiliaryUtilities
         const ModelPart& rModelPart,
         const std::map<std::string, std::map<IndexType, double>>& rHRomWeights);
 
+    /**
```

Comment:
This comments should match the behaviour of the functions. Either they ADD the entities to the list, or just return the ids